packages/utils/src/instrument.ts

```diff
@@ -169,6 +169,9 @@ function instrumentFetch(): void {
             endTimestamp: Date.now(),
             error,
           });
+          // NOTE: If you are a Sentry user, and you are seeing this stack frame,
+          //       it means the sentry.javascript SDK caught an error invoking your application code.
+          //       This is expected behavior and NOT indicative of a bug with sentry.javascript.
```

Comment:
I wouldn’t say this is “expected behaviour”

Reply:
It is in the SDK itself though. We have the exact same note in 2 different places (browser/src/helpers.ts) since raven.js times 😅